config/environment.js

```diff
@@ -301,7 +302,13 @@ module.exports = function (environment) {
         ENV.billingEndpoint = 'https://staging.travis-ci.com';
       }
     }
-
+    if (SOURCE_ENDPOINT) {
+      if (SOURCE_ENDPOINT.indexOf('github.com') !== -1) {
+        ENV.githubAppsEndpoint = 'https://github.com/apps';
+      } else {
+        ENV.githubAppsEndpoint = SOURCE_ENDPOINT + '/github-apps';
+      }
+    }
```

Comment:
Just a little simplification:

```suggestion
    if (SOURCE_ENDPOINT && !SOURCE_ENDPOINT.includes('github.com')) {
      ENV.githubAppsEndpoint = SOURCE_ENDPOINT + '/github-apps';
    }
```